core/arch/arm/kernel/boot.c

```diff
@@ -103,6 +103,14 @@ __weak void main_secondary_init_gic(void)
 {
 }
 
+/* May be overridden in plat-$(PLATFORM)/main.c */
+__weak unsigned long plat_get_aslr_seed(void)
```

Comment:
Perhaps we could just rename `get_aslr_seed()` into this. That there's a `fdt` argument that might or might not mean anything shouldn't be a problem in platform specific code.

Reply:
I previously tried to provide own `get_aslr_seed()` in platform code.
But this required to mark `get_aslr_seed()` as a `__weak` function in `link_dummies_init.c` to resolve link issues. But you told that I can't do this. I'm talking about this comment: https://github.com/OP-TEE/optee_os/pull/4354#discussion_r565231088

Thus, I added that additional function `plat_get_aslr_seed`

Reply:
I previously tried to provide own `get_aslr_seed()` in platform code.
But this required to mark `get_aslr_seed()` as a `__weak` function in `link_dummies_init.c` to resolve link issues. But you told that I can't do this. I'm talking about this comment: https://github.com/OP-TEE/optee_os/pull/4354#discussion_r565231088

Thus, I added that additional function `plat_get_aslr_seed`

Reply:
Oh, sorry. Find then.